### PR TITLE
More changes from feedback

### PR DESCRIPTION
**Pull request by daltzctr**

### CODE REVIEW

**Comment by bhall-ctre** [comment]
A couple other comments from the previous PR:
- Above `getAppliedUpdateFrequency`, one more we can mention is setUpdateFrequencyForAll
- Also, did we plan for this page to be linked in the sidebar, or just from the changelog link on the home page? Doesn't look like we added it to the toc.

**Comment by daltzctr** [comment]
I didn't plan to add it to the sidebar, but I don't have an opinion either way. I don't think people are going to constantly hop to it.

**Comment by daltzctr** [comment]
>Above getAppliedUpdateFrequency, one more we can mention is setUpdateFrequencyForAll

That's already there.

**Comment by bhall-ctre** [comment]
> That's already there.

Yep, sorry, I didn't see https://github.com/CrossTheRoadElec/PhoenixPro-Documentation/pull/97

**Comment by bhall-ctre** [comment]
Other comments:
- "Additional feedforwards in various controls."
Maybe we should mention what all are available now, something like:
"Phoenix 6 now supports kS, kG, kV, and kA feedforward terms."
- I feel like we should expand "Additional feedforwards" and "Support for roboRIO motion profiles" to a new section. We can use WPILib's `TrapezoidProfile` to demonstrate the new features (i.e. no more WPILib feedforward object)

**Comment by bhall-ctre** [comment]
> I didn't plan to add it to the sidebar, but I don't have an opinion either way. I don't think people are going to constantly hop to it.

I agree that people generally won't need to look at previous years' changelogs at least, but I think it would be useful to have the current season's changelog linked in the sidebar (that's at least how it's done in the WPILib docs).

File: source/docs/yearly-changes/yearly-changelog.rst [hunk 1]
```diff
@@ -117,18 +135,39 @@ New Configs
...
 
 * Orchestra has been ported from v5.
 
   * Now supports multiple devices playing a single track.
+  * Now works when the robot is disabled.
+  * A new control mode ``MusicTone`` has been added and can be used for playing a specific frequency.
 
 * Remote limits have been ported from v5.
-* Additional feedforwards in various controls.
 * Support for roboRIO motion profiles using Velocity/Acceleration setpoints in Position/Velocity controls.
```
Comment (bhall-ctre) [inline source/docs/yearly-changes/yearly-changelog.rst:169]: "Support for roboRIO motion profiles" has also been moved out, so this line can be removed

File: source/docs/yearly-changes/yearly-changelog.rst [hunk 2]
```diff
@@ -117,18 +135,39 @@ New Configs
 
 We've added several new configs. A full list of available configs is available in the ``configs`` (Java, C++, Python) namespace.
 
+Improved Support for roboRIO Motion Profiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Additional support has been added for various feedforward terms (kS, kG, kV and KA). There is now improved integration with roboRIO motion profiling using velocity or acceleration setpoints in various closed loop controls.
```
Comment (bhall-ctre) [inline source/docs/yearly-changes/yearly-changelog.rst:141]: "using velocity or acceleration setpoints in various closed loop controls"
Maybe explicitly mention velocity in position control and acceleration in velocity control?

**Pull request merged**